test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -268,7 +282,7 @@ Date: REGEX(.*)
 06. Statistics
 ==============
 HTTP/1.1 200 OK
-Content-Length: 488
+Content-Length: 51REGEX((3|5))
```

Comment:
This is 513 or 515.
Two bytes extra when timing gives that is takes TEN seconds instead of NINE.
Not dirty at all.
The tests must/should work under valgrind as well ...
A functest that fails under valgrind **fails** to get the leaks reported. *That* is important.
If it takes nine or ten seconds ... Not very important ...

However, if we run the REGEXPECT part under cpp before comparing, this 'shortage' can be avoided.   Still, I find this a little exaggerated ...  The functests aren't there to measure timing, but functionality.